### ISSUE TITLE
JS error when trying to select SqueezeBox

### ISSUE THREAD

**Issue by alexanderadam** [LMS-Community/slimserver#169]
Everytime I try to select the SqueezeBox (`Choose Player` on the top right in the default theme) I'm getting the following JS error.

Chrome:
```
js-main.html?r=1505480690:3564 Uncaught TypeError: Cannot read property 'floating' of undefined
    at L._selectPlayer (js-main.html?r=1505480690:3564)
    at L.<anonymous> (js-main.html?r=1505480690:3597)
    at b (js-main.html?r=1505480690:101)
_selectPlayer @ js-main.html?r=1505480690:3564
(anonymous) @ js-main.html?r=1505480690:3597
b @ js-main.html?r=1505480690:101
```

![image](https://user-images.githubusercontent.com/372620/31536246-3e17bff8-affe-11e7-8b19-94f79fb322ef.png)

Firefox:

```
TypeError: pm is undefined
js-main.html:3564:1
_selectPlayer
http://host:9000/html/js-main.html:3564:1
_switchPlayer/update<
http://host:9000/html/js-main.html:3597:5
b
http://host:9000/html/js-main.html:101:88
```

Both meaning the same and pointing to the snippet derived from [UI.js](https://github.com/Logitech/slimserver/blob/dc0cde0d0cfda7920589b786d8bf956e7bbc50a5/HTML/EN/html/SqueezeJS/UI.js#L1347)

I'm using Logitech Media Server Version `7.9.1 - 1505480690 @ Fri Sep 15 14:05:22 UTC 2017`

PS: don't let you irritate from the screenshot. The Squeezebox was found before.
~~PPS: is there any other possibility to connect SqueezeBox and LMS with each other? Ports are open but Subnets are different. Does it make any difference?~~ Nevermind I found it: On the Squeezebox: `My Music > Switch Library`

**Comment by mherger** [comment]
Can't reproduce this issue. And I actually am confused by your edits... When exactly do the errors occur? When you open that menu? When you click the message?

**Comment by mherger** [comment]
The "404 (Not Found)" errors have been fixed in commit 6823be9b553062d13f3d4bd249c203241bdbe4eb. The other issue I haven't been able to reproduce. Feel free to re-open this issue when you've got more information and updated instructions about how to re-produce. Thanks!
